actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -1012,14 +1012,14 @@ def build_options(selected, options = {})
         end
 
         # Build select option HTML for year.
-        # If <tt>year_format</tt> option is not passed
+        # If <tt>year_format</tt> option is not passed to DateTimeSelector
```

Comment:
Please remove this change since it doesn't make sense. We don't pass this option to a module. The example already explain to who we pass it.

Reply:
I reverted the change.

I think you mean the comment implies the option is passed to `date_select` or `datetime_select`. Is it correct?